node/node.go

```diff
@@ -1026,11 +1022,11 @@ func (node *Node) setupProtocols() error {
 	node.BlockMiningAPI = &blockMiningAPI
 
 	// set up retrieval client and api
-	retapi := retrieval.NewAPI(retrieval.NewClient(node.host, node.blockTime, node.PorcelainAPI))
```

Comment:
🎉

Reply:
Note: `node.host` should be accessed on porcelain too in a perfect world. (just observation not relevant to this PR)